### ISSUE TITLE
IOSDriver is able to find an element with FindElementsByAccessibilityId but not with FindElementByAccessibilityId

### ISSUE THREAD

**Issue by timonHa** [appium/appium#13866]
## The problem
IOSDriver is finds one element with FindElementByAccessibilityId. With method FindElementByAccessibilityId IOSDriver is unable to find the element.

var tests = Driver.FindElementsByAccessibilityId("Btn"); 

var test = Driver.FindElementByAccessibilityId("Btn");

## Environment

* Appium version (or git revision) that exhibits the issue: 1.15.1
* Last Appium version that did not exhibit the issue (if applicable): 1.13
* Desktop OS/version used to run Appium: macOS 10.15.2
* Node.js version (unless using Appium.app|exe): 12.14.1
* Npm or Yarn package manager: 6.13.4
* Mobile platform/version under test: iOS 13.3
* Real device or emulator/simulator: real device
* Appium CLI or Appium.app|exe: Appium.app

## Link to Appium logs

https://gist.github.com/timonHa/27b59a107baa0860f7dc9fb96cc4c267

**Comment by mykola-mokhnach** [comment]
According to the WDA code this might happen if the [exists](https://developer.apple.com/documentation/xctest/xcuielement/1500760-exists?language=objc) property of the found element is falsy. Although, this property is not verified if the query is executed for multiple elements. Could it be the case here? What is the page source output and the device log when both queries are being executed?

**Comment by timonHa** [comment]
Sry, I'm new to testing iOS Devices with Appium.

- How can I ceck the exists property of the found element?

-It is not possible to get the page source output "Cannot get 'xml' source of the current application". Like in #13595. 
upgrading to appium 1.16.0 is not possible, because the version is no longer 
available.

-How can I get the device log?  [Get Logs](http://appium.io/docs/en/commands/session/logs/get-log/)?

**Comment by mykola-mokhnach** [comment]
> upgrading to appium 1.16.0 is not possible, because the version is no longer
available.

It does exist and could be installed via `npm`

Regarding the other questions - feel free to ask them at [Appium forum](discuss.appium.io)
Here is the place for issues.
